framework/signature-inference-scripts/infer-and-annotate.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
```

Comment:
This is not a sh script.  It works when I change "/bin/sh" to "/bin/bash", but when /bin/sh is a symbolic link to dash I get the following errors:
<pre>
$t/checker-framework-mernst/framework/signature-inference-scripts/infer-and-annotate.sh nullness $pl/java/plume.jar `findfile '*.java'`
/home/mernst/research/types/checker-framework-mernst/framework/signature-inference-scripts/infer-and-annotate.sh: 27: /home/mernst/research/types/checker-framework-mernst/framework/signature-inference-scripts/infer-and-annotate.sh: Syntax error: "(" unexpected
</pre>
If I fix that, I get
<pre>
/home/mernst/research/types/checker-framework-mernst/framework/signature-inference-scripts/infer-and-annotate.sh: 33: /home/mernst/research/types/checker-framework-mernst/framework/signature-inference-scripts/infer-and-annotate.sh: Bad substitution
</pre>

Can you remove the bashisms from this script so that it is a sh script?